src/Components/Home/Users.js

```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react'
 import { Link } from 'react-router-dom'
 import PropTypes from 'prop-types'
 import { Chip } from 'primereact/chip'
-import { Avatar } from 'primereact/avatar'
+// import { Avatar } from 'primereact/avatar'
```

Comment:
Please delete unused imports 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix